dashboard/app/views/errors/deprecated_course.html.haml

```diff
@@ -0,0 +1,7 @@
+.not-found-page.deprecated-course-page
+  .error-parent
+    .deprecated-course-child
+      %img{src: '/shared/images/happy_penguin.png', alt: 'A happy cartoon penguin'}
+      %h1= I18n.t('course_deprecated.title')
```

Comment:
Awesome! i18n since the design! If you have not done it, I'd suggest checking the page renders as expected with right-to-left languages

Reply:
I have not, thanks for the suggestion! Will do.

Reply:
Chatted in Slack, Mario confirmed RTL looked good!